app/admin/user.rb

```diff
@@ -4,7 +4,12 @@
 # https://github.com/activeadmin/activeadmin/blob/master/docs/2-resource-customization.md#setting-up-strong-parameters
 #
 # permit_params :list, :of, :attributes, :on, :model
-#
+  index do
+    selectable_column
+    column :email
+    column :admin
+    column :created_at
+  end
```

Comment:
You should add a method call `actions` after your last `column` method call, that will take care of creating a couple links to view, edit and delete a `User` resource.